test/integration/TokenTransfers.spec.ts

```diff
@@ -120,33 +120,31 @@ describe('Token transfers', () => {
         const iconUrl = 'http://a.b.com'
         const TBA_URI = `/${identity1.account.getAddress()}/TBA`
 
-        const createToken = new RadixTransactionBuilder().createTokenMultiIssuance(
+        await new RadixTransactionBuilder().createTokenMultiIssuance(
             identity1.account,
             name,
             symbol,
             description,
             granularity,
             amount,
             iconUrl,
-        ).signAndSubmit(identity1).toPromise()
-
-        await createToken
-
-        expect(identity1.account.transferSystem.tokenUnitsBalance[TBA_URI].toString()).to.eq('500')
-        expect(account2.transferSystem.tokenUnitsBalance[TBA_URI]).to.be.undefined
-
-        const mintAndTransfer = new RadixTransactionBuilder().mintTokens(
+        ).mintTokens(
             identity1.account,
             TBA_URI,
-            4000,
+            4000
```

Comment:
This is missing the trailing comma, as tslint will warn you about

The reason we use this rule is that it creates cleaner git diffs and potentially simplifies merges https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Reply:
Fixed. Also added linting pre-commit hook.